tools/video_tools/pyscenedetect_shot_detection.xml

```diff
@@ -0,0 +1,20 @@
+<tool id="PySceneDetect_shot_detection" name="PySceneDetect" version="1.0.0">
```

Comment:
I would recomment to use only small letters as tool-ids

Reply:
And define an explicit dependency on python with the dependencies tag.

Reply:
Pretty much all our tools so far are written in python 3. Is there a way to define such dependency for all tools? 

I'm not sure how Galaxy may automatically resolve dependencies. So far we rely on manual installation on all dependencies. This is of course not ideal. Suggestion would be appreciated!

Reply:
Sure. This is super easy. Just add:

```
    <requirements>
        <requirement type="package" version="3.8">python</requirement>
    </requirements>
```

If you want to share common prams or features of tools please look at macros: https://planemo.readthedocs.io/en/latest/writing_standalone.html#macros